amalgamate.sh

```diff
@@ -11,4 +11,5 @@ make mod_{indexfile,dirlisting,staticfile}
 ./amalgamate.mjs | "${SHELL}" -euox pipefail
 c2rust transpile --overwrite-existing --emit-build-files --binary lighttpd_amalgamated --output-dir ${rust_dir} amalgamated.compile_commands.json
 cp {rust,${rust_dir}}/build.rs
+sed -i "s|${PWD}/||g" ${rust_dir}/src/lighttpd_amalgamated.rs
```

Comment:
please add a comment and link to a c2rust issue at least asking if a feature exists to emit relative paths

Reply:
That's not something `c2rust` really has control over.  This comes from the expansion of the special `__FILE__` macro, which is something `clang`'s `cpp` does.  It may be possible for `c2rust transpile` to intercept these and turn them into `std::file!`, and there may be an issue for that, but there's nothing `c2rust` can do about absolute vs. relative paths.  It's probably caused by the commands in `compile_commands.json`, which are usually generated to be absolute since `clang` has bugs when there are relative paths in a compilation database.